Model/lib/rng/wdkModel-config.rng

```diff
@@ -6,6 +6,8 @@
             <attribute name="modelName" />
             <attribute name="webServiceUrl" />
             <attribute name="smtpServer" />
+            <attribute name="smtpUsername" />
```

Comment:
You need to make these optional or the existing templates will fail.

Reply:
Oops, should be fixed now. There were a couple of other bugs that I dredged up with testing.